Arm/Insts/DPI/Logical_imm.lean

```diff
@@ -61,7 +61,7 @@ def exec_logical_imm (inst : Logical_imm_cls) (s : ArmState) : ArmState :=
 ----------------------------------------------------------------------
 
 /-- Generate random instructions of the DPI.Logical_imm class. -/
-partial def Logical_imm_cls.rand : IO (Option (BitVec 32)) := do
+partial def Logical_imm_cls.dpi.rand : IO (Option (BitVec 32)) := do
```

Comment:
Same suggestion as for `Add_sub_imm` and for the same reason: `.dpi` -> `.inst`

Reply:
Updated. Sorry I missed some comments in the last commit.